app/models/rubric_review.py

```
@@ -0,0 +1,41 @@
+    description: str = Field(..., description="Detailed explanation of the weakness and why it matters.")
+    severity: WeaknessSeverity = Field(..., description="Severity level of the weakness.")
+    suggestion: str = Field(..., description="Actionable recommendation to fix or improve this weakness.")
+    question_index: Optional[int] = Field(
+        None,
+        description="If the weakness is specific to a sub-rubric, include the question index. Omit for global issues.",
+    )
+
+
+class RubricCritique(BaseModel):
+    overall_score: int = Field(
+        ..., ge=0, le=100, description="Quality score for the rubric on a 0–100 scale."
```

Ruff (0.14.3)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)